docs/en/index.rst

```diff
@@ -50,6 +50,12 @@ Configuration
     // Before loading DebugKit
     Configure::write('DebugKit.forceEnable', true);
 
+* ``DebugKit.ignorePaths`` - Regex pattern (including delimiter) to ignore paths.
+   DebugKit won't save data for request URLs which match this regex. Defaults to ``null``::
```

Comment:
```suggestion
   DebugKit won't save data for request URLs that match this regex. Defaults to ``null``::
```
https://www.grammarly.com/blog/which-vs-that/ (not to be pretentious, it's just a good explanation of why `which` feels wrong)

Reply:
I actually appreciate the English grammar lesson :slightly_smiling_face:.